src/core_codemods/docs/pixee_python_flask-enable-csrf-protection.md

```diff
@@ -0,0 +1,10 @@
+Flask views using `FlaskForm` have CSRF protection enabled by default. However other views may use AJAX to perform unsafe HTTP methods. FlaskWTF provides a way to enable CSRF protection globally for all views of a Flask app.
```

Comment:
This is a case where I think we need a fairly detailed description to educate the user on CSRF vulnerabilities and the benefits of this change. I think we also need a pretty detailed description of the risks of such a change.